src/ConcatenateMsg.chpl

```diff
@@ -55,8 +55,7 @@ module ConcatenateMsg
             var valSize: int;
             select objtype {
                 when "str" {
-                    var legacy_placeholder: string;
-                    (name, legacy_placeholder) = rawName.splitMsgToTuple('+', 2);
+                    name = rawName;
```

Comment:
I think it's cleaner to just change the looping variable from `rawName` to `name` (on line 53) to be something like
```C
for (name, i) in zip(names, 1..) {
```
and avoid the extra variable

Reply:
Extra var has been removed.